ibc-testkit/src/testapp/ibc/clients/mock/client_state.rs

```diff
@@ -91,15 +91,13 @@ impl MockClientState {
 impl Protobuf<RawMockClientState> for MockClientState {}
 
 impl TryFrom<RawMockClientState> for MockClientState {
-    type Error = ClientError;
+    type Error = DecodingError;
```

Comment:
Just flagging that there are still several `TryFrom` conversions left that need to be updated to use `DecodingError` as their `Error` type, if we're planning to address them in this PR. You can find them by searching for `type Error =` in your IDE.

Reply:
Cleaned up all of the remaining ones except for those conversion mentioned in #1349.